Client/Frontend/Browser/Tab.swift

```diff
@@ -178,6 +178,14 @@ class Tab: NSObject {
   var isDesktopSite: Bool {
     webView?.customUserAgent?.lowercased().contains("mobile") == false
   }
+  
+  var containsWebPage: Bool {
+    if let url = url {
+      return url.isWebPage() && !(InternalURL(url)?.isAboutHomeURL ?? false)
```

Comment:
`!(InternalURL(url)?.isAboutHomeURL ?? false)` this is very confusing to read haha, maybe we can do something like

```swift
let isHomeURL = InternalURL(url)?.isAboutHomeURL
return url.isWebPage && isHomeURL != true
```

Reply:
But you like the optional dance in Swift :P Fixed at https://github.com/brave/brave-ios/pull/5236/commits/013a9ecefd1849141e901f1d9fb053db9ca938cf